DashAI/back/api/api_v1/endpoints/dataloaders.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from fastapi import APIRouter, status
+from fastapi.exceptions import HTTPException
+
+from DashAI.back.core.config import dataloader_registry
+
+logging.basicConfig(level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.get("/{task_name}")
+async def get_dataloaders(task_name: str):
+    """
+    Returns all the available dataloaders and its schemas.
+    Returns
+    -------
+    List[JSON]
+        List of dataloaders JSONs
+    """
+
+    register_dataloaders = dataloader_registry.task_to_components(task_name)
+    if not register_dataloaders:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND, detail="No dataloaders found"
+        )
```

Comment:
We agreed with the frontend to return an empty list `[]` when there is no object found in the database. Maybe in this case return an empty json `{}`.